eventually/static/src/containers/registerLogin/SetNewPassword.js

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import {withRouter} from 'react-router-dom';
+import TextField from 'material-ui/TextField';
+import RaisedButton from 'material-ui/RaisedButton';
+import {putNewPasswordService} from './registrationService.js';
+
+const style = {
+    margin: 11,
```

Comment:
why 11?

Reply:
yes, it should be 12px. The same as in Login , Register and ForgetPassword style.